### ISSUE TITLE
check cpu and memory utilization

### ISSUE THREAD

**Issue by sabarishsubramanian** [nwoodthorpe/Load-Balancer-Golang#16]
Hello all

I want something where I can calculate the memory and cpu utilisation happening at each http  server by the request . I saw something called profiling in go.  Can u suggest me some examples of it if that's how it's done or is there something else which can do this

**Comment by d12** [comment]
Hey @sabarishsubramanian ,

This package does not offer built-in monitoring for the servers. Is there a reason you want the load balancer to monitor the servers? Unless it's used in distribution of requests, I don't think the load balancer needs to care how much RAM or CPU each server is using. 

This load balancer also doesn't require that servers are on the same machine. In the case of a web server on another machine in the same local network, the load balancer cannot fetch usage data, the web server process or a monitoring process on the server needs to do that.

If your web server processes are local, you can do something like this: https://unix.stackexchange.com/questions/554/how-to-monitor-cpu-memory-usage-of-a-single-process

**Comment by sabarishsubramanian** [comment]
Yes am trying to send request to my load balancer . So I wanted to know how
much each request take memory and cpu utilisation at the server. So that I
can send to my load balancer at regular intervals about the info.

Can someone please help me out.

**Comment by d12** [comment]
Do the answers here help? https://unix.stackexchange.com/questions/554/how-to-monitor-cpu-memory-usage-of-a-single-process

**Comment by sabarishsubramanian** [comment]
Here I can just get to know but I want it as a script so that I can feed to
load balancer
